.github/workflows/test-all.yml

```diff
@@ -25,6 +25,7 @@ env:
   # the beahvior to use the new URLs.
   CHROMEDRIVER_CDNURL: https://googlechromelabs.github.io/
   CHROMEDRIVER_CDNBINARIESURL: https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/
+  CHROME_VALIDATED_VERSION: linux-120.0.6099.71
```

Comment:
Is this possible that this line can be automatically updated?

Reply:
Not easily, unless you have any ideas!